include/swift/AST/DiagnosticsSIL.def

```diff
@@ -953,10 +953,10 @@ ERROR(regionbasedisolation_unknown_pattern, none,
 //
 
 ERROR(regionbasedisolation_transfer_yields_race_no_isolation, none,
-      "transferring value of non-Sendable type %0; later accesses could race",
+      "sending value of non-Sendable type %0; later accesses could race",
```

Comment:
`sending value of non-Sendable type %0` is a little confusing because there are cases where this is okay. I think the second part of the message is the explanation, we should change it to: 

`sending value of non-Sendable type %0 with later accesses risks a data race`